src/LinearFractional.jl

```diff
@@ -97,7 +97,7 @@ mutable struct LinearFractionalModel <: AbstractModel
     end
 end
 
-function LinearFractionalModel(optimizer::Union{Type{<:MOI.AbstractOptimizer}, MOI.OptimizerWithAttributes};
+function LinearFractionalModel(optimizer;
```

Comment:
`::Union{Type{<:MOI.AbstractOptimizer}, MOI.OptimizerWithAttributes, Function should work}`

Reply:
Thanks for the suggestion @blegat.  I think if there is no official requirement for the supertype, I'll just go with dropping the type-specification altogether as @nignatiadis suggested rather than making the type-union even more difficult for a reader to understand.